site/layouts/initiatives.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint import/no-default-export: off */
+import { formatDate } from "@/lib/formatDate.js";
+import allPersons from "../.contentlayer/generated/Person/_index.json";
+
+export default function InitiativeLayout({ children, frontMatter }) {
+  const { title, image, description, homepage, status, resolution, team, alumni, created, url_path } = frontMatter;
+  
+  const homepageUrl = homepage?.startsWith("http") ? homepage : `https://lifeitself.org/initiatives${homepage}`
+  const contributors = allPersons.filter(person => team.concat(alumni).includes(person.id))
+
+  // index page
+  if (url_path === "initiatives") return (
```

Comment:
What are we doing here - this seems quite ugly. Why would we need to apply this layout for the index page?